### DISCUSSION TITLE
Command in "init" array vehicle doesn't repeat when respawn

### DISCUSSION THREAD

**Discussion by YasArma** [Vdauphin/HeartsAndMinds#1412] [Question and answer] [answered]
Hello, Sorry in advance, I already did a research in this repository and none come up with precise problem as mine, 
Wanted to make custom texture on a chopper that respawn on destroyed, after respawn it came with original texture, any how to make the init command repeat after it respawn ? 

![Arma3_x64 2022-09-27 06-02-21](https://user-images.githubusercontent.com/114446077/192389444-b6fb087b-a666-44af-bf25-18c4b6aff5d1.png)

Thank You in advance.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

I thought it was supported by https://community.bistudio.com/wiki/BIS_fnc_getVehicleCustomization which return an array with what they call "texture"
It used here [`btc_veh_fnc_propertiesGet`](https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/veh/propertiesGet.sqf#L34) and called when the heli is added to the respawn system https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/veh/addRespawn.sqf#L47
And restored on respawn here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/veh/respawn.sqf#L65

But it only support texture already in game or https://community.bistudio.com/wiki/Arma_3:_Vehicle_Customization

Also it is `argb` not `rgb`

Cheers

**Reply by YasArma** [reply]
Thank you for your reply Vdauphin, the explanation really helped me understand more this gamemode,

**Reply by Vdauphin** [reply]
I did a PR for this https://github.com/Vdauphin/HeartsAndMinds/pull/1414 in the dev branch
